.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
please revert this for concistency, you can add yours in your own system global gitignore

Reply:
Removed